lib/MediaConnection/RTCPeerMediaConnection.js

```diff
@@ -0,0 +1,155 @@
+const EventEmitter = require('events').EventEmitter;
+
+module.exports = class RTCPeerMediaConnection extends EventEmitter {
+
+    constructor(pcConfig, iceConnectionStateCallback)
+    {
+        super();
+        this.rtcPeerConnection = new RTCPeerConnection(pcConfig);
+        this.rtcPeerConnection.addEventListener('iceconnectionstatechange', () => {
+            iceConnectionStateCallback(this.rtcPeerConnection.iceConnectionState)
+        });
+    }
+
+
+    createSdpOffer(rtcSessionDescription) {
+
+        return this.rtcPeerConnection.signalingState === 'stable' ? this.rtcPeerConnection.createOffer(rtcSessionDescription)
+            .then(offer => {
+                return this._setLocalSdp(offer);
+            }) : Promise.resolve();
+    }
+
+    createSdpAnswer(rtcSessionDescription) {
+
+        return this.rtcPeerConnection.createAnswer(rtcSessionDescription).then(offer => {
```

Comment:
Same here, `createSdpAnswer()` must return a SDP.

Reply:
I reworked that part a bit.